src/exercises/day-1/luhn.rs

```diff
@@ -15,25 +15,29 @@
 // ANCHOR: solution
 // ANCHOR: luhn
 pub fn luhn(cc_number: &str) -> bool {
-    // ANCHOR_END: luhn
-    let mut digits_seen = 0;
+    let cc_number = cc_number.trim().replace(" ", "");
```

Comment:
Is this necessary? I would expect the `filter` call below to take of skipping the spaces?

I think it's more elegant to filter below since it avoids an allocation: `str::replace` will heap-alloacte space for the result, but filtering the existing string is an in-place operation.